src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -445,6 +498,7 @@ public void WriteTo(ObjectWriter writer)
                 writer.WriteString(Language);
                 writer.WriteString(FilePath);
                 writer.WriteString(OutputFilePath);
+                writer.WriteString(OutputRefFilePath);
```

Comment:
@heejaechang Does this need a bump of some version number somewhere?

Reply:
no, the one who persist this data need to. right now, there is no one who persist this data.

Reply:
excellent -- thanks for checking @heejaechang!